configs/projects/bolt-runtime.rb

```diff
@@ -130,17 +130,55 @@
   proj.component 'rubygem-hiera-eyaml'
 
   # Core dependencies
+  proj.component 'rubygem-addressable'
+  proj.component 'rubygem-bindata'
+  proj.component 'rubygem-builder'
+  proj.component 'rubygem-CFPropertyList'
+  proj.component 'rubygem-colored'
+  proj.component 'rubygem-concurrent-ruby'
+  proj.component 'rubygem-connection_pool'
+  proj.component 'rubygem-cri'
+  proj.component 'rubygem-docker-api'
+  proj.component 'rubygem-erubis'
+  proj.component 'rubygem-excon'
+  proj.component 'rubygem-facter'
+  proj.component 'rubygem-faraday'
+  proj.component 'rubygem-faraday_middleware'
   proj.component 'rubygem-ffi'
+  proj.component 'rubygem-gssapi'
+  proj.component 'rubygem-gyoku'
+  proj.component 'rubygem-hiera'
+  proj.component 'rubygem-httpclient'
+  proj.component 'rubygem-little-plugger'
+  proj.component 'rubygem-log4r'
+  proj.component 'rubygem-logging'
   proj.component 'rubygem-minitar'
   proj.component 'rubygem-multi_json'
```

Comment:
I noticed that even though multi_json is being built here, it seems like bolt-vanagon has also been building its own copy. bolt-vanagon's version of this gem is 0.7, while the version built here is 0.8. Will updating that version be ok?